Dockerfile

```diff
@@ -21,4 +21,11 @@ RUN wget -q https://github.com/AsamK/signal-cli/releases/download/v${SIGNAL_CLI_
     ln -sf /opt/signal-cli-${SIGNAL_CLI_VERSION}/bin/signal-cli /usr/local/bin/
 
 WORKDIR /app
-COPY scripts/register-number.sh /app
\ No newline at end of file
+COPY scripts/register-number.sh /app
+
+
+COPY . /app
+RUN pip install .
+
+CMD ["python", "signal_scanner_bot/bin/run.py"]
```

Comment:
Since you are pip installing above this could technically be `CMD ["signal-scanner-bot"]` -- your preference though.

Reply:
I tried this command and still couldn't manage to get it to work :thinking: . I plan on investigating/improving the Dockerfile in #3 - this "worked" and I was able to get a version working with it at least.